src/Symfony/Component/Notifier/Bridge/Clickatell/ClickatellOptions.php

```diff
@@ -1,61 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Notifier\Bridge\Clickatell;
-
-use Symfony\Component\Notifier\Message\MessageOptionsInterface;
-
-/**
- * @author gnito-org <https://github.com/gnito-org>
- */
-final class ClickatellOptions implements MessageOptionsInterface
```

Comment:
Why removing?

Reply:
Ah because it does not really have an options.

We can remove it because it was merged and not released right now, correct?

Reply:
Correct